### ISSUE TITLE
pq: column tl.alphanumcurrency does not exist

### ISSUE THREAD

**Issue by FredericHeem** [stellar/go-horizon#56]
Here is the error message while trying to get the master account info:

```
INFO[0010] Starting request                              method=GET path=/accounts/GCEZWKCA5VLDNRLN3RPRJMRZOX3Z6G5CHCGSNFHEYVXM3XOJMDS674JZ
DEBU[0010] Negotiated content type                       accept=text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8 content_type=application/hal+json
INFO[0010] Executing query                               sql=SELECT ha.* FROM history_accounts ha WHERE address = $1 LIMIT 1
INFO[0010] Executing query                               sql=SELECT a.accountid, a.balance, a.seqnum, a.numsubentries, a.inflationdest, a.thresholds, a.flags FROM accounts a WHERE accountid = $1 LIMIT 1
INFO[0010] Executing query                               sql=SELECT tl.accountid, tl.issuer, tl.alphanumcurrency, tl.tlimit, tl.balance, tl.flags FROM trustlines tl WHERE accountid = $1
ERRO[0010] pq: column tl.alphanumcurrency does not exist  file=src/github.com/stellar/go-horizon/render/problem/main.go line=65
```

**Comment by FredericHeem** [comment]
fixed in PR https://github.com/stellar/go-horizon/pull/57

**Comment by nullstyle** [comment]
was fixed in #62
